projects/aca-shared/src/lib/components/page-layout/page-layout.component.scss

```diff
@@ -53,3 +53,13 @@
     border-right: 1px solid var(--theme-border-color, rgba(0, 0, 0, 0.07));
   }
 }
+
+@media screen and (max-width: 380px) {
```

Comment:
can you put it inside .aca-page-layout selector because ViewEncapsulation.None so to avoid that it will affect on other components.

Reply:
yeah sure I have now added media selector inside .aca-page-layout selector :)